python/test/test_polars.py

```diff
@@ -118,7 +118,7 @@ def test_private_lazyframe_explicit_sum(measure):
             pl.Series("A", [10.0] * 5),
         ]
     )
-    df_act = m_lf(lf).sort("B").collect()
+    df_act = m_lf(lf).collect()
```

Comment:
Is it wrong to `sort` as a post processing step? Or maybe it just exercises more of the machinery to do it here? If it's not a good thing to do, that should get mentioned in the docs. (Though maybe it's already there?)

Reply:
The output of the measurement is a OnceFrame now, which has a more limited API. One of the additions in this PR is to allow sorting within `make_private_lazyframe`. Operations that are on what are now OnceFrames have been moved into the constructor.